src/routes/articles.js

```diff
@@ -39,10 +39,49 @@ const createArticle = async (req, res, next) => {
 	}
 }
 
+// GET POST BY ID
+const getArticle = async (req, res, next) => {
+	const { id } = req.params
+	try {
+		const article = await getPost({
+			id
+		})
+		if (!article) {
+			return res.status(404).json({
+				success: false,
+				message: 'Article does not exist'
+			})
+		}
+		return res.status(200).json({
+			status: 'success',
+			data: {
+				articleId: article.id,
```

Comment:
as at now we have 2 article responses being returned from this file now and they are basically the same except the `comments` property.

Time to get one place to transform the article response

This is to make sure when something has to be changed in the response, you don't have to update `getArticle` and `createArticle` at the same time (or  sometimes, understandably forget to update one of them)

Reply:
for now, kindly create a function called `transformArticleResponse` in the same file. This function will take the response from the service (getPost, createPost functions in the service) and do the transformation once

at the end your code for both `getArticle` and `createArticle` should look like

```js
return res.status(200 /*or 201 */).json({
  status: 'success',
  data: transformArticleResponse(article)
})
```

Reply:
yes please

Reply:
I have pushed the edits now @jocrah